KIPs/kip-113.md

```diff
@@ -83,6 +83,45 @@ After reading from IKIP113, users must perform the following validations:
 - [KeyValidate](https://www.ietf.org/archive/id/draft-irtf-cfrg-bls-signature-05.html#section-2.5)
 - [PopVerify](https://www.ietf.org/archive/id/draft-irtf-cfrg-bls-signature-05.html#section-3.3.3)
 
+### JSON-RPC API
+
+The following JSON-RPC method for the Klaytn node should be added to provide the registered bls public keys.
```

Comment:
```suggestion
The following JSON-RPC method for the Klaytn node should be added to provide the registered BLS public keys.
```

Reply:
Applied in https://github.com/klaytn/kips/pull/159/commits/3364e36fd1e3ffdd671530229053a8928459f08f